src/Mod/Part/App/TopoShape.h

```diff
@@ -925,6 +925,9 @@ class PartExport TopoShape: public Data::ComplexGeoData
     void mapSubElement(const TopoShape &other,const char *op=nullptr, bool forceHasher=false);
     void mapSubElement(const std::vector<TopoShape> &shapes, const char *op=nullptr);
     void mapSubElementsTo(std::vector<TopoShape>& shapes, const char* op = nullptr) const;
+    void flushElementMap() const;

```

Comment:
I think the lint errors above this point can be ignored, although that last one points at an error I must have made and will correct.   Those errors are waiting for an overall cleanup pass on this file.

Reply:
Ok, I'll leave this as is for now.